ocaml/xapi/xapi_pool_update.ml

```diff
@@ -170,7 +171,7 @@ let create_yum_config ~__context ~self ~url =
   let signed = String.length key <> 0 in
   let signed_index = if signed then 1 else 0 in
   let name_label = Db.Pool_update.get_name_label ~__context ~self in
-  (Printf.sprintf ("[main]\nkeepcache=0\nreposdir=/dev/null\ngpgcheck=%d\nrepo_gpgcheck=%d\n\n") signed_index signed_index)
+  (Printf.sprintf ("[main]\nkeepcache=0\nreposdir=/dev/null\ngpgcheck=%d\nrepo_gpgcheck=%d\ninstallonlypkgs=\n\n") signed_index signed_index)
```

Comment:
I'd like to propose to organise this slightly different:


    let config =
      String.concat "\n"
      [ "[main]"
      ; "keepcache=0"
      ; "reposdir=/dev/null"
      ; Printf.sprintf "gpgcheck=%d" signed_index
      ; Printf.sprintf "installonlypkgs=%s" ....
      ]

This keeps the line structure of the file you want to create and you still have the flexibility to use `sprintf` for individual lines.